packages/web/src/types.ts

```diff
@@ -158,9 +159,18 @@ interface SignWithLsig {
 	args?: Uint8Array[];
 }
 
+interface SignWithMultisig {
+	sign: SignType.MultiSignature;
+	mparams: WalletMultisigMetadata;
+	fromAccount?: AccountSDK;
+	fromAccountAddr: AccountAddress;
+}
+
 export type Lsig = SignWithLsig;
 
-export type Sign = SignWithSk | SignWithLsig;
+export type MultiSig = SignWithMultisig;
```

Comment:
let's don't create a type alias. We can export `SignWithMultisig` directly

Reply:
Done